### DISCUSSION TITLE
Docusaurus

### DISCUSSION THREAD

**Discussion by TheKnarf** [tinacms/tinacms#3839] [Ideas]
I'm using Docusaurus for internal company documentation, I would love to make it easier for people to edit it (I'm essentially using it as an alternative to a wiki like Confluence). Would it be possible to integrate TinaCMS and Docusaurus?

**Comment by logan-anderson** [comment]
@TheKnarf We have a [blog post](https://tina.io/blog/tinasaurus-docusaurus-starter/) with an example of using TInaCMS and Docusaurus

**Reply by erdavnet** [reply]
I've followed the blog post and successfully integrated TinaCMS with Docusaurus. However, I’m currently struggling with adding authentication to protect the CMS dashboard

Is there any guidance or workaround for adding custom auth (like Clerk, Auth.js, or even basic credentials) to TinaCMS when used in a static site like Docusaurus?

**Reply by TheKnarf** [reply]
Stick it in an Nginx Docker container and add Basic auth to the nginx config.
